Source/WebCore/page/Frame.h

```diff
@@ -90,6 +91,7 @@ class Frame : public ThreadSafeRefCounted<Frame, WTF::DestructionThread::Main>,
     virtual void setOpener(Frame*) = 0;
     virtual const Frame* opener() const = 0;
     virtual Frame* opener() = 0;
+    virtual FrameLoaderClient& loaderClient() = 0;
```

Comment:
This could be a covariant virtual function, then `WebFrame::localFrameLoaderClient()` could call this instead of `loader().client()`, but it's probably fine either way.

Reply:
That is true, but there are so many places where we call LocalFrame::loader().client() that this keeps that consistent.